### ISSUE TITLE
[Violation] 'setTimeout' handler took <N>ms

### ISSUE THREAD

**Issue by sergiocormio** [olefirenko/vue-barcode-reader#18]
This error is shown on chrome console at DEMO page and on my application:
Chrome Version 93.0.4577.63 
```
BrowserCodeReader.js?46a5:490 Trying to play video that is already playing.
eval @ BrowserCodeReader.js?46a5:490
step @ BrowserCodeReader.js?46a5:32
eval @ BrowserCodeReader.js?46a5:13
eval @ BrowserCodeReader.js?46a5:7
__awaiter @ BrowserCodeReader.js?46a5:3
BrowserCodeReader.tryPlayVideo @ BrowserCodeReader.js?46a5:484
videoCanPlayListener @ BrowserCodeReader.js?46a5:466
147[Violation] 'setTimeout' handler took <N>ms
BrowserCodeReader.js?46a5:820 [Violation] 'setTimeout' handler took 72ms
BrowserCodeReader.js?46a5:820 [Violation] 'setTimeout' handler took 64ms
BrowserCodeReader.js?46a5:820 [Violation] 'setTimeout' handler took 59ms
BrowserCodeReader.js?46a5:820 [Violation] 'setTimeout' handler took 58ms
...
```
Thanks!

**Comment by Cristian-Ayala** [comment]
Read [this](https://stackoverflow.com/questions/59063979/violation-settimeout-handler-took-nms) related problem posted on stackoverflow.

TLDR:
Is taking "too long" for it to feel like a responsive application. You don't have to fix this for it to work, if it is working well enough for you. It is trying to provide a profiling of places you might improve your app / code. This may come from "@zxing/library".
